tests/contrib/django/test_restframework.py

```diff
@@ -0,0 +1,53 @@
+import django
+from django.apps import apps
+from nose.tools import ok_, eq_
+from unittest import skipIf
+
+from .utils import DjangoTraceTestCase
+
+@skipIf(django.VERSION < (1, 10), 'requires django version >= 1.10')
+class RestFrameworkTest(DjangoTraceTestCase):
+    def setUp(self):
+        super(RestFrameworkTest, self).setUp()
+
+        # We do the imports here because importing rest_framework with an older version of Django 
+        # would raise an exception
+        from rest_framework.views import APIView
+        from ddtrace.contrib.django.restframework import ORIGINAL_HANDLE_EXCEPTION, unpatch_rest_framework
+
+        self.APIView = APIView
+        self.ORIGINAL_HANDLE_EXCEPTION = ORIGINAL_HANDLE_EXCEPTION
+        self.unpatch_rest_framework = unpatch_rest_framework
+
+    def test_setup(self):
+        ok_(apps.is_installed('rest_framework'))
+        ok_(hasattr(self.APIView, self.ORIGINAL_HANDLE_EXCEPTION))
+
+    def test_unpatch(self):
+        self.unpatch_rest_framework()
+        ok_(not hasattr(self.APIView, self.ORIGINAL_HANDLE_EXCEPTION))
+
+    def test_tracer(self):
```

Comment:
What is the purpose of that test ?

Reply:
Testing if the tracer is enabled. Do you think I should remove it?

Reply:
Did your code change the logic for that? Because otherwise I think it's already checked elsewhere in the tests.